### PR TITLE
Up to go 1.6.2 for build and test.

### PR DESCRIPTION
**Pull request by spxtr**
~~1.6.1 contains some security fixes. 1.6.2 should be out soon.~~ 1.6.2 is out :D

Images aren't pushed yet.

### CODE REVIEW

**Comment by ixdy** [comment]
Should we also update references to `kubekins-test` to use `:0.11`?

**Comment by spxtr** [comment]
Yes, thank you. Also upped travis version.

**Comment by ixdy** [comment]
Note #23931 is also updating kube-cross. Not sure which of these PRs should take priority.

cc @luxas

**Comment by luxas** [comment]
@spxtr Please don't push images before `lgtm` on the PR to avoid conflicts
You'll have to repush `v1.6.1-1` after #23931
I was actually wondering earlier if I should bump to `go1.6.1` in #23931, but came to the conclusion that it should be made in separate PRs

**Comment by luxas** [comment]
@spxtr Please move `all: push` to the top of the `build/build-image/cross/Makefile`, which makes `make` == `make push`
ref: https://github.com/kubernetes/kubernetes/pull/23931#issuecomment-210120719

**Comment by ixdy** [comment]
Should probably update `hack/jenkins/dockerized-e2e-runner.sh` too.

**Comment by spxtr** [comment]
> Should probably update `hack/jenkins/dockerized-e2e-runner.sh` too.

It might be worth using a `VERSION` file that all these places read from like the cross image does.

**Comment by ixdy** [comment]
#23931 won the race, so this needs to be rebased.

**Comment by spxtr** [comment]
@ixdy PTAL. I'll need to repush the build image since some changes went in to the dockerfile.

**Comment by ixdy** [comment]
Still don't see an update to `hack/jenkins/dockerized-e2e-runner.sh`. Using a `VERSION` file would be fine, too.

**Comment by spxtr** [comment]
Added dockerized-e2e-runner. We don't actually have the source handy when we curl that down on Jenkins, so a VERSION file would also have to be curled down which I'm not a huge fan of.

**Comment by luxas** [comment]
fyi, `go1.6.2` is out already

**Comment by spxtr** [comment]
Hmm, they haven't pushed the 1.6.2 docker image yet...

**Comment by ixdy** [comment]
Looks like the `golang:1.6.2` image exists now.

**Comment by spxtr** [comment]
Yay, alright I'll take care of that.

File: hack/jenkins/test-image/Makefile [hunk 1]
```diff
@@ -14,7 +14,7 @@
 
 all: push
 
-TAG = 0.9
+TAG = 0.11
```
Comment (ixdy) [inline hack/jenkins/test-image/Makefile]: I assume you force-pushed a new version?

Comment (spxtr) [inline hack/jenkins/test-image/Makefile]: Yes.

**Comment by spxtr** [comment]
@ixdy PTAL

**Pull request merged**